content/applications/inventory_and_mrp/inventory/routes/strategies/storage_category.rst

```diff
@@ -0,0 +1,171 @@
+   assigned to.
+
+Capacity by weight
+------------------
+
+On a storage category form (:menuselection:`Inventory app --> Configuration --> Storage
+Categories`), set a maximum product weight in the :guilabel:`Max Weight` field. This limit applies
+to each location assigned this storage category.
+
+Capacity by product
+-------------------
+
+In the :guilabel:`Capacity by Product` tab, click :guilabel:`Add a Line` button to input items and
+their capacities in the :guilabel:`Quantity` field.
+
+.. example::
+   Ensure only a maximum of five `Large Cabinets` and two `Corner Desk Right Sit` are stored at a
+   single storage location by specifying those amounts in the :guilabel:`Capacity by Product` tab of
+   a storage category form.
+
+   .. image:: storage_category/capacity-by-product.png
+      :align: center
+      :alt: Show storage category limiting by product count.
+
+.. _inventory/routes/set-capacity-package:
+
+Capacity by package
+-------------------
+
+For companies using :doc:`packages <../../management/products/usage>` it becomes possible to ensure
+real-time storage capacity checks based on package types: e.g., crates, bins, boxes.
+
+.. important::
+   Enable the :guilabel:`Packages` feature in :menuselection:`Inventory app --> Configuration -->
+   Settings` to show the :guilabel:`Capacity by Package` tab.
+
+.. example::
+   Create putaway rules for pallet-stored items by creating the `High Frequency pallets` storage
+   category.
+
+   In the :guilabel:`Capacity by Package` tab, specify the number of packages for the designated
+   :guilabel:`Package Type` and set a maximum of `2.00` `Pallets` for a specific location.
+
+   .. image:: storage_category/storage-category.png
+      :align: center
+      :alt: Create a storage category on the page.
+
+.. _inventory/routes/assign-location:
+
+Assign to location
+==================
+
+Once the storage category is created, assign it to a location. Navigate to the location by going to
+:menuselection:`Inventory app --> Configuration --> Locations`, and select the location. Click
+:guilabel:`Edit` and select the created category in the :guilabel:`Storage Category` field.
+
+.. example::
+   Assign the `High Frequency pallets` storage category (which limits pallets stored at any location
+   to two pallets) to the `WH/Stock/pallets/PAL 1` sub-location.
+
+   .. image:: storage_category/location-storage-category.png
+      :align: center
+      :alt: When a Storage Category is created, it can be linked to a warehouse location.
+
+.. _inventory/routes/set-putaway-attribute:
+
+Putaway rule
+============
+
+With the :ref:`storage category <inventory/routes/define-storage>` and :ref:`location
+<inventory/routes/assign-location>` set up, create the :doc:`putaway rule <putaway>` by navigating
+to :menuselection:`Inventory app --> Configuration --> Putaway Rules`.
+
+Click the :guilabel:`Create` button to create the putaway rule. In the :guilabel:`Having Category`
```

Comment:
```suggestion
Click the :guilabel:`New` button to create the putaway rule. In the :guilabel:`Having Category`
```

Reply:
Since this is 15.0, the button still says Create. It'll be different in 16.0+ though!